src/components/portfolio/PortfolioOverview.test.tsx

```diff
@@ -22,33 +24,22 @@ describe('PortfolioOverview', () => {
     };
   };
 
-  test('show message if no portfolios exist', () => {
-    const { container, getByText } = renderComponent({ portfolios: [] });
-    expect(getByText('portfolio.dashboard.title')).toBeInTheDocument();
-    expect(getByText('portfolio.dashboard.noPortfolios')).toBeInTheDocument();
-    // no table
-    expect(container.querySelector('table')).toBeNull();
-    // no view selection
-    expect(container.querySelectorAll('button svg')).toHaveLength(0);
-  });
-
   test('switch between table and cards view', () => {
-    // TODO: Update test once cards view is implemented
     const { container, queryByText } = renderComponent();
     expect(queryByText('portfolio.dashboard.title')).toBeInTheDocument();
     expect(queryByText('portfolio.dashboard.noPortfolios')).toBeNull();
-    expect(queryByText('cards')).toBeNull();
     expect(container.querySelectorAll('table')).toHaveLength(1);
+    expect(container.querySelectorAll('cards')).toHaveLength(0);
```

Comment:
This query selects `<cards>` elements. Such an element will never exist.
```suggestion
    expect(container.querySelectorAll('ul')).toHaveLength(0);
```